src/items/rules/identify_creature.json

```diff
@@ -0,0 +1,47 @@
+  },
+  "pages": [
+    {
+      "_id": "G3J86W5JwqmT8B19",
+      "name": "Identify Creatures",
+      "type": "text",
+      "_stats": {
+        "compendiumSource": null,
+        "coreVersion": "12.331",
+        "createdTime": null,
+        "duplicateSource": null,
+        "lastModifiedBy": "Ze895EIq7nm5kMrK",
+        "modifiedTime": 1734732996525,
+        "systemId": "sfrpg",
+        "systemVersion": "0.27.0"
+      },
+      "flags": {},
+      "image": {},
+      "ownership": {
+        "default": -1,
+        "l0dJClEFprrDW4fl": 3
+      },
+      "sort": 100000,
+      "src": null,
+      "system": {},
+      "text": {
+        "content": "<p><strong>Source</strong> Starfinder Core Rulebook pg. 132</p><p>You can use certain skills to identify creatures. The skill used to identify each creature type is listed below and in the individual skill descriptions. A successful skill check allows you to recall a useful piece of information about a specific creature, such as its special powers or vulnerabilities. For every 5 points by which the result of your check exceeds the DC, you recall another piece of useful information. You can attempt untrained skill checks to identify a creature if the DC is 10 or less. You can take 20 on a check to identify a creature, but only if you have a means of performing research, such as access to an information network like a planetary infosphere or a downloaded data set (see page 430); this typically takes 2 minutes.</p><p>The DCs for skill checks to identify creatures are based on the creature’s rarity.</p><table><tbody><tr><td><p>Creature Rarity</p></td><td><p>DC</p></td></tr><tr><td><p>Very common (space goblin)</p></td><td><p>5 + 1-1/2 × creature’s CR</p></td></tr><tr><td><p>Average (most monsters)</p></td><td><p>10 + 1-1/2 × creature’s CR</p></td></tr><tr><td><p>Rare (novaspawn)</p></td><td><p>15 + 1-1/2 × creature’s CR</p></td></tr></tbody></table><p>The list of creature types below indicates which skill is used to identify each creature type.</p><table><tbody><tr><td><p>Creature Type</p></td><td><p>Skill</p></td></tr><tr><td><p>Aberration</p></td><td><p>Life Science</p></td></tr><tr><td><p>Animal</p></td><td><p>Life Science</p></td></tr><tr><td><p>Construct (magical)</p></td><td><p>Mysticism</p></td></tr><tr><td><p>Construct (technological)</p></td><td><p>Engineering</p></td></tr><tr><td><p>Dragon</p></td><td><p>Mysticism</p></td></tr><tr><td><p>Fey</p></td><td><p>Mysticism</p></td></tr><tr><td><p>Humanoid</p></td><td><p>Life Science</p></td></tr><tr><td><p>Magical beast</p></td><td><p>Mysticism</p></td></tr><tr><td><p>Monstrous humanoid</p></td><td><p>Life Science</p></td></tr><tr><td><p>Ooze</p></td><td><p>Life Science</p></td></tr><tr><td><p>Outsider</p></td><td><p>Mysticism</p></td></tr><tr><td><p>Plant</p></td><td><p>Life Science</p></td></tr><tr><td><p>Undead</p></td><td><p>Mysticism</p></td></tr><tr><td><p>Vermin</p></td><td><p>Life Science</p></td></tr></tbody></table>",
```

Comment:
Add a title for each table as a heading, e.g. "Identify Creature by Rarity".
![Example](image.png)

Reply:
Also add inline checks for skills in the second table

Reply:
Realized my added section was already in Skills.  Deleted new add and applied your suggested changes to the existing Skills journal entry.